### ISSUE TITLE
Define and conduct a V&V Plan with Github Actions CI/CD pipeline for eoAPI

### ISSUE THREAD

**Issue by Schpidi** [EOEPCA/data-access#79]
- [ ] Identify validation actions for OGC API Coverages

**Comment by jonas-eberle** [comment]
@j08lue @jankovicgd Is this for OGC API Maps? At the moment we do not have a WCS/OGC API Coverages service available.

**Comment by j08lue** [comment]
Yes, I think booking some of our advances on OGC API Maps towards this task would be appropriate, since the pygeoapi + eoAPI work may also pave the way for coverages, if we decide to do that later.

@Schpidi, please close this ticket as completed or move it to the next work order, as you see fit.

**Comment by jankovicgd** [comment]
Research in eoapi helm chart: 3 hours
